send/crocs_user_export.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import sys
+import os.path
+import send_lib
+from datetime import datetime
+import re
+
+SERVICE_NAME = "crocs_user_export"
+
+TIMEOUT = "5400" #90s * 60 sec = 1.5h
+TIMEOUT_KILL = "60" # 60 sec to kill after timeout
+
+send_lib.check_input_fields(sys.argv)
+
+facility = sys.argv[1]
+destination = sys.argv[2]
+destination_type = send_lib.DESTINATION_TYPE_SERVICE_SPECIFIC if len(sys.argv) < 4 else sys.argv[3]
+
+service_files_dir = send_lib.get_gen_folder(facility, SERVICE_NAME)
+
+#unless specific configuration for destination exists use common configuration for all destination
+service_files_for_destination = service_files_dir + "/_destination/" + destination
+
+if not os.path.isdir(service_files_for_destination):
+	service_files_for_destination = service_files_dir + "/_destination/all"
+
+send_lib.check_destination_type_allowed(destination_type, send_lib.DESTINATION_TYPE_SERVICE_SPECIFIC)
+send_lib.check_destination_format(destination, destination_type)
```

Comment:
I would probably move these checks to right after line 17 (to keep the argument retrieving/validating logic coupled together)